examples/raytrace-parallel/.gitignore

```diff
@@ -1,2 +1 @@
-raytrace_parallel.js
-raytrace_parallel_bg.wasm
+snippets
```

Comment:
I thought now that everything is in `pkg` we don't need the `.gitignore` at all?

Reply:
We still generate a `snippets/worker.js` file which is not in `pkg`. I'm not sure why it's there or how to move it into `pkg`. Do you not see this when you run `build.sh` or `build.py`?

Reply:
Nope, for me the snippets folder lands in the `pkg` directory, as it should. Could you double check?

Reply:
Oh, yes it is in `pkg` now!